tests/conftest.py

```diff
@@ -3163,11 +3175,14 @@ def log_alerts(request):
     """
     teardown = config.RUN["cli_params"].get("teardown")
     dev_mode = config.RUN["cli_params"].get("dev_mode")
+    import_without_odf = config.RUN["cli_params"].get("import-without-odf")
```

Comment:
Why not go with ocsci_config.RUN["import_without_odf"] like on other places?  ANd if you go with cli_params are you sure about `import-without-odf` shouldnt this be: `import_without_odf`? I am not sure about it hence would rather go with the same approach as on other places.